style.css

```diff
@@ -0,0 +1,62 @@
+body{
+    margin : 0;
+}
+.navigation {
+    display: flex;
+    flex-direction: row;
+   
```

Comment:
Empty line :) 
Check the code below for not needed empty lines.
I personally  do one empty line as separator between css classes. This increase readability of the code.